sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 5.1.0-beta.4 (unreleased)
+## 5.1.0-beta.5 (unreleased)
```

Comment:
The bot should take care of adding this after release, so we don't publish this "unreleased" line with the new package.